dist/videojs-transcript.js

```diff
@@ -246,17 +247,21 @@ var scrollerProto = function(plugin) {
       var elementOffsetBottom = element.offsetTop + element.clientHeight;
       var relTop = element.offsetTop - parent.offsetTop;
       var relBottom = (element.offsetTop + element.clientHeight) - parent.offsetTop;
+      var centerPosCorrection;
       var newPos;
 
+      if (plugin.settings.scrollToCenter){
+        centerPosCorrection = Math.round(parent.clientHeight/2 - element.clientHeight/2);
+      }
```

Comment:
So what about the case `plugin.settings.scrollToCenter == false`?
What's `centerPosCorrection` is going to be?